Formula/thors-sql.rb

```diff
@@ -0,0 +1,58 @@
+class ThorsSql < Formula
+  desc "SQL library for C++14"
+  homepage "https://github.com/Loki-Astari/ThorsSQL"
+  url "https://github.com/Loki-Astari/ThorsSQL.git",
+      :tag => "1.5.6",
+      :revision => "2da4ab817a4f45f0d3f8a7bf7ad0fc455031f0f6"
+
+  depends_on "mysql" => :build
+  depends_on "openssl"
```

Comment:
Is there any configuration that can be used to specify which openSSL to use? Otherwise it might pick up Apples SSL.

Reply:
It links against: -lcrypto
It includes:        <openssl/sha.h>